geometry/distance.c

```diff
@@ -0,0 +1,37 @@
+#include <math.h>
+#include <stdio.h>
```

Comment:
Please add a one-line description of what the library/header is for (**see the example below**).

```c
#include <stdio.h>      /// for IO operations
#include <assert.h>    /// for assert
```